llm_toolkit/corpus_generator.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Corpus generator using LLMs."""
+
+import os
+
+from data_prep import introspector
+from experiment import oss_fuzz_checkout
+from experiment.benchmark import Benchmark
+from llm_toolkit import models
+from llm_toolkit import output_parser as parser
+from llm_toolkit import prompt_builder
+
+
+def get_script(
+    ai_binary: str,
+    fixer_model_name: str,
+    target_harness_path: str,
+    benchmark: Benchmark,
+) -> str:
+  """Uses LLMs to generate a python script that will create a seed corpus for a
+  harness.
+
+  The script generated is purely generated and should be considered untrusted
+  in the general sense. OSS-Fuzz-gen already executes arbitrary code since
+  OSS-Fuzz-gen executes arbitrary open source projects with no checking on
+  what code is committed to the given projects."""
```

Comment:
Did you mean "OSS-Fuzz-gen already executes arbitrary code from arbitrary open source projects with no checking on what code is committed to the given projects"?